website/src/documentation/sdks/python-dependencies.md

```diff
@@ -26,460 +26,41 @@ behavior in the service. If you are using any of these packages in your code, be
 aware that some libraries are not forward-compatible and you may need to pin to
 the listed versions that will be in scope during execution.
 
-<p>To see the compile and runtime dependencies for your Beam SDK version, expand
-the relevant section below.</p>
+Dependencies for your Beam SDK version are listed in `setup.py` in the Beam repository. To view them, perform the following steps:
 
-<details><summary markdown="span"><b>2.17.0</b></summary>
+1. Open `setup.py`.
```

Comment:
Can we link the file to the latest file?

```
[`setup.py`](https://raw.githubusercontent.com/apache/beam/v{{ site.release_latest }}/sdks/python/setup.py)
```

Reply:
The example in the remarks for this step now shows and links to the latest version of the file.